.github/workflows/continous-integration.yml

```diff
@@ -134,6 +135,7 @@ jobs:
       DJANGO_SECRET_KEY: secret
       DOMAIN_REDIRECT_MIDDLEWARE_ENABLED: False
       GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      NEWSLETTER_SIGNUP_METHOD: "BASKET"
```

Comment:
Letting the CI newsletter tests use Basket for now to avoid overcomplicating things.